elements/k-new-post.html

```diff
@@ -24,68 +51,106 @@
       }
 
       .preview {
+        padding-left: 15px;
+        padding-right: 15px;
         font-size: 18px;
         line-height: 1.4em;
       }
     </style>
+    <iron-media-query query="(min-width: 752px)" query-matches="{{wide}}"></iron-media-query>
 
     <paper-tabs selected="{{selected}}">
       <paper-tab>Edit</paper-tab>
       <paper-tab>Preview</paper-tab>
     </paper-tabs>
 
-    <iron-pages selected="{{selected}}">
+    <iron-pages selected="{{selected}}" wide-layout$="{{wide}}">
       <div>
-        <k-title title="{{titleValue}}"></k-title>
-        <k-short-content short-content="{{shortContentValue}}"></k-short-content>
-        <k-category category="{{categoryValue}}" categories="[[categoryList]]"></k-category>
-        <k-image-upload
-          image="{{imageValue}}"
-          image-upload-url="[[headerUploadUrl]]"
-          method="[[uploadMethod]]"
-          thumbnail="{{thumbnailImage}}"
-          image="{{imageValue}}"
-        ></k-image-upload>
-
-        <iron-image
-          src$="[[thumbnailImage]]"
-          preload
-          fade
-        ></iron-image>
-
-        <k-editor content="{{contentValue}}"></k-editor>
-        <br/>
-      </div>
+        <template is="dom-if" if="{{wide}}">
+          <div class="container flex-horizontal">
+            <div class="flex8child">
+              <k-title title="{{titleValue}}"></k-title>
+              <k-editor content="{{contentValue}}"></k-editor>
+            </div>
+            <div class="flex4child">
+              <k-category category="{{categoryValue}}" categories="[[categoryList]]"></k-category>
+              <k-short-content short-content="{{shortContentValue}}"></k-short-content>
+              <k-image-upload
+                class="image-upload"
```

Comment:
see the comment above, can simply use k-image-upload to define CSS rule, no need to assign a class